pkgs/stdenv/generic/setup.sh

```diff
@@ -120,15 +120,15 @@ exitHandler() {
     exitCode="$?"
     set +e
 
-    if [ -n "${showBuildStats:-}" ]; then
-        times > "$NIX_BUILD_TOP/.times"
-        local -a times=($(cat "$NIX_BUILD_TOP/.times"))
+    if [[ -n "${showBuildStats:-}" ]]; then
+        local -a buildTimesArray
+        IFS=" " read -r -a buildTimesArray < "$NIX_BUILD_TOP/.times"
```

Comment:
Hmm, have you just removed the line that created the `.times` file?

Reply:
I did make a mistake here.
I was originally thinking about combining those steps to not create the file anymore. (I wanted to put the the times command directly as an input). Now that i think about it though, it might not be good to get rid of that file.
My original thought was that since it's echoed, perhaps it's not needed to keep a record.
I'll add it back just in case it is used actually.

Reply:
I have no strong opinion on keeping the file, but in case of debugging functionality I would indeed err on the side of giving more ways to find data.